.github/workflows/latest-kernel.yml

```diff
@@ -0,0 +1,43 @@
+name: Test build latest kernel available in archlinux
```

Comment:
This is awesome! Wonder if there's some way that we could get active notifications from the scheduled job, say as notifications to a slack channel for example. If we can't get active notifications we might end up not paying attention to it and having it silently fail in the background.

Reply:
I am not sure :/ for now, i think it's ok to check latest master CI result!

Reply:
That's fine, it just feel like extra work because someone needs to actively go check the scheduled run on the `Actions` section (AFAIK the schedule event doesn't get reflected on the master branch check)